apps/trustlab/src/payload/access/roles.js

```diff
@@ -0,0 +1,9 @@
+export const ROLE_ADMIN = "administrator";
+export const ROLE_EDITOR = "editor";
+export const ROLE_AUTHOR = "author";
+export const ROLE_DEFAULT = ROLE_AUTHOR;
```

Comment:
I'm not sure we need this... why not make sure a role is specified wherever/whenever it's needed?